tests/unit/db.test.ts

```diff
@@ -0,0 +1,13 @@
+import { readdirSync } from 'fs';
+import { getMigrationVersion } from '../../db/schemaMigrations';
+
+test('migration files', () => {
+  const migrations = readdirSync('../db/migrations');
+
+  const invalidFile = migrations.find(file => !getMigrationVersion(file));
+  if (invalidFile) {
+    throw new Error(
+      `Migration file - ${invalidFile} has incorrect pattern. Migration file should always follow the pattern 'V{version}_{description}.sql'. For example 'V1_add_new_entity.sql'`
```

Comment:
is "version" going to be 1, 2, 3? Version doesn't seem like the right word to me, to me I think version is more like the difference between "old" (V1) and "new" (V2), where as each migration shouldn't have multiple versions of it. 

Maybe just a number instead of letter prefix? e.g.
```suggestion
      `Migration file - ${invalidFile} has incorrect pattern. Migration file should always follow the pattern '{id}_{description}.sql'. For example '1_add_new_entity.sql'`
```

Reply:
I know lots of tools will use a dev timestamp to avoid conflicts, but we need to ensure a strict order, so number like you have makes more sense, but they can be hard to coordinate with a big team or long lived PRs

Reply:
I tried to match the naming pattern of how flyway does it, but then that has a pattern of V1__example_file.sql, where prefix can be a V, U or R ([link](https://www.red-gate.com/blog/database-devops/flyway-naming-patterns-matter)). Anyways, I was OK with removing the prefix V and sticking to sequential numbering. 

> I know lots of tools will use a dev timestamp to avoid conflicts, but we need to ensure a strict order, so number like you have makes more sense, but they can be hard to coordinate with a big team or long lived PRs

I think that is a good problem to have. I'll be happy to change when there are 15-20 people working on SQ with new features all the times 😄